EasyRestProjectNetTeam2/EasyRestPages/BasePage.cs

```diff
@@ -35,11 +35,21 @@ public void WaitElementIsEnable(int timeToWait, IWebElement element)
             wait.Until(driver => element.Enabled);
         }
 
+        public void WaitElementIsClickable(int timeToWait, IWebElement element)
```

Comment:
That method could NOT be public because it has the `IWebElement` object as a parameter. Please consider the correct access modifier of that method or just do not pass IWebElement as a parameter.
@Slim1011